lib/pbench/cli/agent/commands/results/push.py

```diff
@@ -27,15 +28,24 @@ def execute(self) -> int:
         res = crt.copy_result_tb(
             self.context.token, self.context.access, self.context.metadata
         )
-        if 200 <= res.status_code <= 201:
-            click.echo(f"Status: {res.status_code}, reason: {res.reason}, message: {res.message}")
+
+        try:
+            msg = res.json()["message"]
+        except requests.exceptions.JSONDecodeError:
+            msg = "FOO"
+
+        if res.ok:
+            click.echo(
+                f"Status: {res.status_code}, reason: {res.reason}, message: {msg}"
+            )
```

Comment:
I'm still not convinced we want to write status information to `stdout` on success. This would happen for each `push` operation on top of the summary `move` message. Messy?

Reply:
OK - I'll do as Webb suggests: return 0 for res.ok(), but print the dup message in the 200 case.

Reply:
I wouldn't worry about 200 unless we want `pbench-results-move` or the passthrough server to do something special with that, and I'm not entirely sure they should care (it's just an expensive idempotent operation). The critical piece would be how we tell them, e.g. with an alternate success exit status, which is a kinda flaky concept. If we're not distinguishing on return, then logging a message just seems like noise.